app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
- [ ] Good job, because the name of your foreign key in the schema is author_id, kindly add `foreign_key: 'author_id'` to this line of code to tell rails not to search for default user_id in your post table but instead use author_id, so your code will look like this `has_many :posts, foreign_key: 'author_id'`